server/server_test.go

```diff
@@ -291,6 +291,9 @@ func TestHandleCert(t *testing.T) {
 	s, addr := setupServer(www)
 	defer s.Close()
 
+	// TODO(banaag): Insert 5 second delay to test flake theory.
```

Comment:
You mean "TODO: Replace 5 second delay with something more precise once flake theory is confirmed."?